src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/controller/GeneralPartnerController.java

```diff
@@ -163,19 +158,4 @@ public ResponseEntity<Object> deleteGeneralPartner(@RequestAttribute(TRANSACTION
 
         return new ResponseEntity<>(HttpStatus.NO_CONTENT);
     }
-
```

Comment:
With removal of these three cost end-points in the controllers, they should also be removed from the `InterceptorConfig` class `COST_ENDPOINTS` array:

            "/transactions/*/limited-partnership/partnership/*/costs",
            "/transactions/*/limited-partnership/general-partner/*/costs",
            "/transactions/*/limited-partnership/limited-partner/*/costs"